lib/mutant/cli.rb

```diff
@@ -155,6 +155,9 @@ def add_filter_options(opts)
       opts.on('--ignore-subject EXPRESSION', 'Ignore subjects that match EXPRESSION as prefix') do |pattern|
         add_matcher(:ignore_expressions, config.expression_parser.(pattern))
       end
+      opts.on('--since REVISION', 'Only select subjects touched since REVISION') do |revision|
```

Comment:
> Is revision the best name for this?

Yes.

The `git log` manpage makes it clear:

```
SYNOPSIS
       git log [<options>] [<revision range>] [[--] <path>...]
```

I think `revision` in the git domain can be sloppily defined as: 

Any string git can parse into an unique commit. Including tags.

This can be confirmed via the `git rev-parse` manpage:

```
SPECIFYING REVISIONS
       A revision parameter <rev> typically, but not necessarily, names a commit object. It uses what is called an extended SHA-1 syntax. Here are various ways to spell object names. The ones listed near the end of this
       list name trees and blobs contained in a commit.
```

>  I've seen you use the branch name, and I assume the SHA works. Does a tag work too?

Yes. Even relative expressions such as `HEAD^1` work or more obscure ones.